doc/pages/command-parsing.asciidoc

```diff
@@ -1,26 +1,24 @@
 = Command Parsing
 
-Kakoune commands, either loaded from a script, or written in the command
-prompt are parsed according to the following rules:
+Kakoune commands, either loaded from a script or written in the command prompt, are parsed according to the following rules:
 
 == Basic parsing
 
-- Commands are separated by `;` or end of lines
+- Commands are delimited by a `;` or an end of line
```

Comment:
Not sure if "delimited" is fully correct, because the last command does not need a delimiter.

Reply:
I would think there must be something about the parsing mechanism that identifies a command as a command. To do this would entail there being conditions for such an identification. Such conditions (whitespace before the command, an an end line, or return key?) would serve to delimit the argument, at least in the ordinary sense of that term.